crates/matrix-sdk-ui/src/timeline/traits.rs

```diff
@@ -113,7 +109,7 @@ impl RoomDataProvider for Room {
 
         Some(Profile {
             display_name: latest_event.sender_display_name().map(ToOwned::to_owned),
-            display_name_ambiguous: latest_event.sender_name_ambiguous().unwrap_or(false),
+            display_name_ambiguous: latest_event.sender_name_ambiguous().unwrap_or_default(),
```

Comment:
I kinda like having `unwrap_or(false)`, because it tells you what the expected type is here? :pleading_face:

Reply:
It's also shorter. I don't see how `unwrap_or_default` is better for `bool`s:

Reply:
OK OK! ;-)